ci/bamboo/build-windows.ps1

```diff
@@ -0,0 +1,227 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+# Builds and tests the nupic.bindings python extension using Release build
+# type.
+#
+# ASUMPTIONS:
+#          1. Building via Numenta Vagrant Task using Vagrant image
+#             bamboo-vagrant-windows-1.x.x.
+#
+#          2. nupic.core root is the current directory on entry.
+#
+#          3. Expects a pristine nupic.core source tree without any remnant
+#             build artifacts from prior build attempts. Otherwise, the
+#             behavior is undefined.
+#
+# NOTES: Based on examples in comments of
```

Comment:
Please remove this. The destination isn't public and even if it was, we should document what is necessary here rather than relying on an old PR.